tools/requirements.txt

```diff
@@ -1,8 +0,0 @@
-mariadb
-gitpython
-pylint
-pyyaml
-colorama
-zmq
-black
-regex
```

Comment:
Why was this deleted?

Reply:
Cause I wasn't paying attention and it wasn't on ignore.  Will fix. :)

Reply:
This is still being deleted and it's probably the cause of the CI fail

Reply:
Yeah, not sure if I am gonna be able to fix it retroactively, its on my ignore now but still showing, I'll probably just rebranch and resubmit once I resolve everything here.